internal/pkg/skuba/upgrade/addon/versions.go

```diff
@@ -80,13 +87,16 @@ func PrintAddonUpdates(updatedAddons AddonVersionInfoUpdate) {
 		if updatedAddons.Current[addon].Version != updatedAddons.Updated[addon].Version {
 			fmt.Printf("  - %s: %s -> %s\n", addon, updatedAddons.Current[addon].Version, updatedAddons.Updated[addon].Version)
 		} else {
-			fmt.Printf("  - %s: %s -> %s (manifest version from %d to %d)\n", addon,
-				updatedAddons.Current[addon].Version, updatedAddons.Updated[addon].Version,
-				updatedAddons.Current[addon].ManifestVersion, updatedAddons.Updated[addon].ManifestVersion)
+			if len(updatedAddons.Current[addon].Version) > 0 && len(updatedAddons.Updated[addon].Version) > 0 {
```

Comment:
I find this nested `if .. else .. if .. else` hard to read and understand , is not possible to write it differently?
maybe using `continue`  instead of `else` in L89?